setup.py

```diff
@@ -44,7 +44,7 @@ def get_packages(package):
 env_marker_below_38 = "python_version < '3.8'"
 
 minimal_requirements = [
-    "click==7.*",
+    "click==8.*",
```

Comment:
Why not relax just the upper bound ?

Reply:
@euri10 done. I have made it >=7.*

Reply:
![image](https://user-images.githubusercontent.com/3006502/118900511-c0403680-b92e-11eb-9fdf-0cd6ffc88ba0.png)